tests/core/test_logger.py

```diff
@@ -14,8 +16,8 @@
 @pytest.fixture(scope="session")
 def webtestapp():
     Config._config = None
-    setup_logging('tests/resources/test.ini#main')
-    app = get_app('tests/resources/test.ini#main')
+    setup_logging('tests/resources/test.ini#main', global_conf=os.environ)
+    app = get_app('tests/resources/test.ini#main', options=os.environ)
```

Comment:
hello,
I have huge trouble running tests in a local environment. If there are escaped environment variables (%s) this command fails.
I am testing some stuff, but I would recommend extracting the required values from os.environ like
```
app = get_app('tests/resources/test.ini#main', options={'PGHOST':os.environ.get('PGHOST', 'oereb-db')
```

Reply:
I have a local environment variable LESSOPEN='| /usr/bin/lesspipe %s'
this variable being present in the environment messes up the get_app command